modules/cudaimgproc/include/opencv2/cudaimgproc.hpp

```diff
@@ -800,10 +800,21 @@ enum MomentsOrder {
 @param order Order of largest moments to calculate with lower order moments requiring less computation.
 @returns number of image moments.
 
-@sa cuda::moments, cuda::spatialMoments, cuda::MomentsOrder
+@sa cuda::spatialMoments, cuda::moments, cuda::MomentsOrder
  */
 CV_EXPORTS_W int numMoments(const MomentsOrder order);
 
+/** @brief Converts the spatial image moments returned from cuda::spatialMoments to cv::Moments.
+@param spatialMoments Spatial moments returned from cuda::spatialMoments.
+@param order Order used when calculating image moments with cuda::spatialMoments.
+@param momentsType Precision used when calculating image moments with cuda::spatialMoments.
+
+@returns cv::Moments.
+
+@sa cuda::spatialMoments, cuda::moments, cuda::cvtToMoments, cuda::numMoments, cuda::MomentsOrder
+ */
+CV_EXPORTS_W Moments cvtToMoments(Mat spatialMoments, const MomentsOrder order, const int momentsType);
```

Comment:
cvt->convert or, better, `convertSpatialMoments`. Current name does not describe the function behaviour.